contribute/index.md

```diff
@@ -54,3 +54,59 @@ The question is: do you want to ...
   - submit a use case
   - answer questions on Slack and Discourse
   - contribute something that isn't addressed in this guide!
+
+## How do I find contributing opportunities?
+
+If by this point you are convinced that contributing is for you, but don't know where and how to find contributing opportunities, we have collated a couple of ideas for you. For each idea we have provided a list of *prerequisites* and *estimated time* to make the contribution.
+
+
+### Find a README file to edit
+
+There is plenty of documentation files in the Julia-verse of GitHub, and typos are so easy to make! You can help cleaning up documentation by correcting typos or errors which you find. 
+
+- pre-requisites: GitHub account, text editor with spell checking (e.g. MS Word, Google docs)
+- estimated time: 1h
+- steps:
+	- [pick a Julia package](#how-to-find-julia-packages)
+	- find a typo to fix (e.g. paste the text into Google docs for a spell-check)
+	- go through a few green buttons on GitHub to submit your contribution
+	
+### Find questions to answer: StackOverflow
+- pre-requisites: you do not need an acocunt to find a question, but you will need a StackOverflow account to answer one
```

Comment:
Great idea to insert a typo here and below to give a chance to practice the previous paragraph right away :D

Reply:
That's the idea :)